AdyenEncryption/AnyEncryptor.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Defines the interface to encrypt given data with JWE Encryption.
+internal protocol AnyEncryptor {
+    static func encrypt(_ payload: Payload, with publicKey: String) throws -> String
```

Comment:
no need to expose this in protocol interface, its supposed to be a helper function and not supposed to be overridden